pipeline/recon/config.py

```diff
@@ -24,12 +25,15 @@
     "subjack": f"{Path.home()}/go/bin/subjack",
     "subjack-fingerprints": f"{Path.home()}/go/src/github.com/haccer/subjack/fingerprints.json",
     "gobuster": f"{Path.home()}/go/bin/gobuster",
-    "recursive-gobuster": f"{defaults.get('tools-dir')}/recursive-gobuster/recursive-gobuster.pyz",
+    "recursive-gobuster": f"{defaults.get('tools-dir')}/recursive-gobuster/",  # recursive-gobuster.pyz -- tool filename for ref
```

Comment:
this change will break scanning with `--recursive`.

Most of the tool definitions are paths to the tool themselves in order to execute them as commands.  Example [here](https://github.com/epi052/recon-pipeline/blob/master/pipeline/recon/web/gobuster.py#L141)